stable/admin/templates/statefulset.yaml

```diff
@@ -50,6 +50,15 @@ spec:
           mountPath: /var/opt/nuodb
         - name: log-volume
           mountPath: /var/log/nuodb
+      - name: init-raft
+        image: {{ template "init.image" . }}
+        imagePullPolicy: {{ default "" .Values.busybox.image.pullPolicy }}
+        command: ['rm' , '-rf', '/var/opt/nuodb/raftlog' ]
```

Comment:
Good find.
This is meant to be:
a) conditional on the admin.persistence.initialize flag;
b) a one-off event when restarting the admin layer - and then disabled again;

There is no easy way to delete the raftlog, and there are multiple recovery scenarios where that is needed.

However, I'll remove this from this PR, and handle it as a separate branch and PR, since criteria b) above requires a better solution.